task3/runner.py

```diff
@@ -0,0 +1,12 @@
+"""Functions runner module."""
+import pytasks
+
+
+def runner(*funcs):
+    """Run functions implemented in pytasks module."""
+    for func in funcs:
+        print(func())
+
+
+# example
+runner(pytasks.generate_numbers, pytasks.fizzbuzz)
```

Comment:
This is a good and graceful solution but we shall improve it a bit cause accoring to our task condition we should send function names: runner(‘generate_numbers’) – print result  only for generate_numbers(), not functions.